README.md

```diff
@@ -25,6 +25,10 @@
 - Run `./build.sh`. If it doesn't run, make sure it is executable.
 > To view the program as doc, run : `godoc -http=:6060` and checkout at `http://localhost:6060/pkg/kwoc-backend/`
 
+### Setting up pre-commit hook (optional)
+- Check if `golangci-lint` is installed, if not, install using `go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.53.3`
+- Run `git config core.hooksPath .githooks` (make sure it is executable using `chmod +x .githooks/*`)
```

Comment:
Are there instructions for Windows? Also if possible, please add a link to the documentation for this git config variable.

Reply:
the hooks are run by git using the shell emulator shipped with git for windows anyway, so it works even when you commit from command prompt (used powershell to commit, worked).

Reply:
and for windows, the instructions are the same, except chmod, which shouldn't be required anyway because I pushed as executable

Reply:
okay